utensor_cgen/operators.py

```diff
@@ -131,6 +131,7 @@ class OperatorFactory():
                 "Dequantize": _DequantizeOperator,
                 "Max": _MaxOperator,
                 "Min": _MinOperator,
+                "QuantizedAdd", _AddOperator,
```

Comment:
It's a syntax error.
Should be
```
...
"QuantizedAdd": _AddOperator,
...
```

Reply:
In addition to this syntax error, I don't think this hot fix will work anyway.
Since the QuantizedAdd takes 6 inputs but normal AddOp takes only 2 inputs.
Replacing QuantizedAdd with AddOp won't work anyway.

Here is the screen shot of the code generated with your PR, it just push too much inputs to the AddOp in `uTensor`:
<img width="1159" alt="screen shot 2018-02-24 at 2 45 16 pm" src="https://user-images.githubusercontent.com/6830390/36626738-f015ea94-1972-11e8-99da-e77c93fa252e.png">

The definition of AddOp in `uTensor`:
https://github.com/uTensor/uTensor/blob/master/uTensor/ops/MathOps.hpp#L376

Reply:
Before QuantizedAdd is available in `uTensor`, I'd like to recommend users to use a script on `TensorFlow` GitHub repo:
https://raw.githubusercontent.com/tensorflow/tensorflow/master/tensorflow/tools/quantization/quantize_graph.py

This script will not add QuantizedAdd to your graph as `TransformGraph` does.
I know it's not the standard way to do it, but it works fine for now.

Reply:
And this project is aimed to provide tool which can generate a "working" cpp/hpp implementation in `uTensor` for users.
What I mean by "working" is compilable and runnable on a MCU.
As a result, I prefer not to merge the PR.